ddtrace/internal/packages.py

```diff
@@ -112,7 +115,7 @@ def _root_module(path: Path) -> str:
     # Try to resolve the root module using sys.path. We keep the shortest
     # relative path as the one more likely to give us the root module.
     min_relative_path = max_parent_path = None
-    for parent_path in (Path(_).resolve() for _ in sys.path):
```

Comment:
can `sys.path` change at runtime?

Reply:
we could cache on `len(sys.path)` which is not perfect, but probably course grain enough to hit most cases?